assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/embedded-metadata/embedded-metadata-container.styles.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { createStyles } from 'antd-style'
+
+export const useStyles = createStyles(({ css, token }) => {
+  return {
+    tab: css`
+        p:first-of-type {
```

Comment:
Which other parameter should be in that tab and above the headline?
But yeah i will wrap it in a div :)

Reply:
Currently it applies to any first <p> in every element in the tab. 

<div class="i-am-a-grid-cell">
  <p></p>
</div>

Also this would be affected by the styles. I just want to make sure that we don't get any unwanted side effects in that case.

Reply:
ah nice i thought only for the first match - shitty CSS :P

Reply:
wrapped it in a div https://github.com/pimcore/studio-ui-bundle/pull/325/commits/819b265cf13197e379d4c84aa63fb09d73e993be#diff-d18079ee22fd1e056a3e26d670a4d749686da0dc4b25b2a52b29cb2d148a18cfR64